NMS-14245: Add event when device backup starts (#4676)
OpenNMS/opennms: docs/modules/operation/pages/device-config-backup/dcb.adoc

.. In the case of backups that have failed or are in progress, no icon appears.
. To perform actions such as compare, download, and view history, place a check beside the device(s) and click the menu option.

You can search by node label or IP address and sort the list of backups by typing in the search bar and clicking on the headers in the table respectively.

== Monitoring

> Events are sent to indicate when backups are started and completed for each monitored device-config service. All events are marked with the appropriate node and interface information.

Reviewer: ```suggestion
Events are sent to indicate when backups are started and completed for each monitored device-config service.
All events are marked with the appropriate node and interface information.
```

> When a backup is started, an event is sent with the following UEI.

Reviewer: ```suggestion
When a backup starts, an event is sent with the following UEI:
```

    uei.opennms.org/deviceconfig/configBackupStarted

When a backup completes, an event is sent with one of the following UEIs:

    uei.opennms.org/deviceconfig/configBackupSucceeded
    uei.opennms.org/deviceconfig/configBackupFailed

> Additional Parameters

Reviewer: ```suggestion
=== Additional parameters
```

> * *controlProtocol* - The name of the control-plane protocol used by OpenNMS to initiate the backup ("cron", "REST").

Reviewer: ```suggestion
* *controlProtocol* - The name of the control-plane protocol OpenNMS uses to initiate the backup ("cron", "REST").
```

.. In the case of backups that have failed or are in progress, no icon appears.
. To perform actions such as compare, download, and view history, place a check beside the device(s) and click the menu option.

You can search by node label or IP address and sort the list of backups by typing in the search bar and clicking on the headers in the table respectively.

== Monitoring

Events are sent to indicate when backups are started and completed for each monitored device-config service.
All events are marked with the appropriate node and interface information.

> When a backup starts, an event is sent with the following UEI:

Reviewer 2: Events information already described in `docs/modules/operation/pages/device-config-backup/configuration.adoc`
